checkConstraints.py

```diff
@@ -67,37 +68,46 @@ async def checkConstraints(q_id):
 
     async with ClientSession() as session:
         async with session.get(CONSTRAINT_CHECK_URL + '&id=' + q_id) as r:
-            r = await r.read()
-            parsed_response = json.loads(str(r, 'utf-8'))
-
-            for (property_id, statement_group) in parsed_response['wbcheckconstraints'][q_id]['claims'].items():
-                for statement in statement_group:
-                    counter['statement_is_violated'] = False
-
-                    violated_mainsnaks = statement['mainsnak']['results']
-                    for violated_mainsnak in violated_mainsnaks:
-                        # print("property_id, status:", property_id, main_result['status'])
-                        counter = incrementCounter(violated_mainsnak['status'], counter)
-
-                    if 'qualifiers' in statement.keys():
-                        qualifier_items = statement['qualifiers'].items()
-                        for (qualifier_property_id, qualifier_item) in qualifier_items:
-                            for qualifier_constraint_check in qualifier_item:
-                                qualifier_results = qualifier_constraint_check['results']
-                                for qualifier_result in qualifier_results:
-                                    # print("property_id, qualifier_property_id, status:", property_id, qualifier_property_id, qualifier_result['status'])
-                                    counter = incrementCounter(qualifier_result['status'], counter)
-
-                    if 'references' in statement.keys():
-                        reference_items = statement['references']
-                        for reference_item in reference_items:
-                            for (snak_property_id, reference_constraint_checks) in reference_item['snaks'].items():
-                                for reference_constraint_check in reference_constraint_checks:
-                                    reference_results = reference_constraint_check['results']
-                                    for reference_result in reference_results:
-                                        # print("property_id, snak_property_id, status:", property_id, snak_property_id, reference_result['status'])
-                                        counter = incrementCounter(reference_result['status'], counter)
-            return counter
+            try:
```

Comment:
Can the `ClientError` occur in anything but the call to `r.read` directly below this? I don't think it can, thus it should by enough to wrap the `r.read` call in the try-except block.